package.json

```diff
@@ -45,6 +45,20 @@
             "C:\\some-dir\\dprint.exe"
           ]
         },
+        "dprint.command": {
```

Comment:
Could this functionality be built into `dprint.path` instead?

Reply:
probably, but I'm certain it would confuse some, e.g. I wouldn't think that `path` is possibly a `npm run dprint` command